docs/api/js/utils.md

```diff
@@ -38,17 +43,35 @@ export const IERC20 = new utils.Interface(require("../abi/IERC20.json"));
 Utility method that returns the correctly formed `paymasterParams` object for the common [paymaster flows](../../dev/developer-guides/aa.md#built-in-paymaster-flows).
 
 ```typescript
-export function getPaymasterParams(paymasterAddress: Address, paymasterInput: PaymasterInput): PaymasterParams;
+export function getPaymasterParams(paymasterAddress: Address, paymasterInput: PaymasterInput): PaymasterParams {
+    if (paymasterInput.type == 'General') {
+        return {
+            paymaster: paymasterAddress,
+            paymasterInput: getGeneralPaymasterInput(paymasterInput)
+        };
+    } else {
+        return {
+            paymaster: paymasterAddress,
+            paymasterInput: getApprovalBasedPaymasterInput(paymasterInput)
+        };
+    }
+}
 ```
 
 The definition of the `PaymasterInput` can be found [here](./types.md).
 
-## Default pubdata price limit
-
+## Useful gas constants
 Currently, there is no method to accurately estimate the required `gasPerPubdataLimit`. That's why for now, it is highly recommended to provide the `DEFAULT_GAS_PER_PUBDATA_LIMIT`. Users are not charged more by providing it.
 Later on it will be possible to query the current recommended limit.
 
 ```typescript
-const GAS_PER_PUBDATA_BYTE = 16;
-export const DEFAULT_GAS_PER_PUBDATA_LIMIT = GAS_PER_PUBDATA_BYTE * 10_000;
+const GAS_PER_PUBDATA_BYTE = 17;
+
+export const DEFAULT_GAS_PER_PUBDATA_LIMIT = 800;
```

Comment:
Please update according to the latest version of the SDK (a new one was released on Friday)